docs/backup-recovery/README.md

```diff
@@ -0,0 +1,132 @@
+# Cluster upgrades - backup and recovery #
+
+## Background ##
+
+This feature creates a pre-upgade backup and provides a procedure for rapid recovery of a SNO in the event of a failed upgrade that is unrecoverable. In case of a upgrade failure, this feature allows the SNO to be restored to a working state with the previous version of OCP without requiring a re-provision of the application(s).  
```

Comment:
I'd drop the "that is unrecoverable", since we're talking about how to recover.

Reply:
I have rephrased the sentence, does it sound better?